### PR TITLE
Make nbconvert a little less chatty.

### PR DESCRIPTION
**Pull request by Carreau**
User don't really care what template is used by default.
and no need to say where the files might be written if they won't be.

### CODE REVIEW

**Comment by Carreau** [comment]
Fix @jhamrick complaint also. 

BTW I can `--author`the last commit as you if you like :-)

**Comment by jhamrick** [comment]
:+1: 

> BTW I can --author the last commit as you if you like :-)

No, it's fine to leave it as it is.

This will probably conflict with #8073 which I just submitted but not in a major way.

Can I make one other suggestion, which is to make the "Executing notebook with kernel:" message in the execute preprocessor also be debug?

**Comment by Carreau** [comment]
> This will probably conflict with #8073 which I just submitted but not in a major way.

Yes, just one line. I'll rebase this one once yours is merge. 



> Can I make one other suggestion, which is to make the "Executing notebook with kernel:" message in the execute preprocessor also be debug?

I'm torn. Execution can take quite some time, and without the message, people could think the conversion is stuck. I would like to know what others think.

File: IPython/nbconvert/writers/files.py [hunk 1]
```diff
@@ -76,7 +76,11 @@ def write(self, output, resources, notebook_name=None, **kw):
             # Write all of the extracted resources to the destination directory.
             # NOTE: WE WRITE EVERYTHING AS-IF IT'S BINARY.  THE EXTRACT FIG
             # PREPROCESSOR SHOULD HANDLE UNIX/WINDOWS LINE ENDINGS...
-            for filename, data in resources.get('outputs', {}).items():
+
+            items = resources.get('outputs', {}).items()
+            if items:
+                self.log.info("Support files will be in %s", os.path.join(resources['output_files_dir'], ''))
```
Comment (takluyver) [inline IPython/nbconvert/writers/files.py]: Test failure: `KeyError: 'output_files_dir'`

Comment (Carreau) [inline IPython/nbconvert/writers/files.py]: Yep. seen travis angry red mail.

**Comment by jhamrick** [comment]
> Execution can take quite some time, and without the message, people could think the conversion is stuck.

Hrm, good point. Ok, maybe it's fine to leave that in, then.

**Comment by Carreau** [comment]
We can try to have a coroutine, that make  a spinning `/-\|/-\|/-\|/-\|/-\|/` or [others](http://stackoverflow.com/questions/2685435/cooler-ascii-spinners).

**Comment by Carreau** [comment]
[Oh gosh](http://jsfiddle.net/CgkQJ/1160/) !

**Pull request merged**